docs/stdlib/generic.rst

```diff
@@ -270,11 +278,12 @@ Generic
 
     :index: find strpos strstr position array
 
-    A polymorphic function to test if the *haystack* contains the *needle*.
+    Returns :eql:type:`bool` for whether a ``haystack`` set has a ``needle``
```

Comment:
```suggestion
    Returns true if the given sub-value exists within the given value.
```

Keeping `true` as plain text here for consistency with the earlier comparison operators.